.github/actions/veracode/source_clear.sh

```diff
@@ -4,10 +4,7 @@ echo "=========================== Starting SourceClear Script ==================
 PS4="\[\e[35m\]+ \[\e[m\]"
 set +e -v -x
 
-mvn -B -q clean install \
-    ${SRCCLR_INSTALL_DEFAULT_OPTIONS} ${SRCCLR_INSTALL_OPTIONS} \
-    com.srcclr:srcclr-maven-plugin:scan \
-    -Dcom.srcclr.apiToken=${SRCCLR_API_TOKEN} > scan.log
+srcclr scan > scan.log
```

Comment:
maybe now this scan.log trickery is not needed anymore?

I guess it was there because maven output was a lot more verbose before

Reply:
Good point, but I'd actually prefer going the other way. 
I just added the "--loud" option to make the logging a lot more verbose and left the redirect, will be a tad easier to start investigation if something fails.

Reply:
Changed my mind after Domenico's comments, I'll just do your approach. All that the --loud option does is show maven logs, where it fails once, because it tries to get stuff without actually building, then when it sees that it failed, it builds the project and uploads that. Having these fails might actually be more confusing than not having logs for maven at all.